bin/build_pgmath.sh

```diff
@@ -41,7 +41,7 @@ else
    fi
 fi
 
-COMP_INC_DIR=$(ls -d $AOMP_INSTALL_DIR/lib/clang/*/include )
+COMP_INC_DIR=$(ls -d $OUT_DIR/lib/clang/*/include )
```

Comment:
This is still not correct, needs llvm in the path:
$OUT_DIR/llvm/lib/clang/*/include